config/v1/0000_10_config-operator_01_infrastructure-TechPreviewNoUpgrade.crd.yaml

```diff
@@ -558,9 +558,38 @@ spec:
                             type: string
                           maxItems: 2
                           type: array
+                        loadBalancer:
```

Comment:
don't look at the CRD, it's not generated from the latest API but from a previous patchset.